EventKit/eventkit-cloud
# Update Load script with topics and attribute classes.

# Description of Improvement

Update tests scripts to make topics and attribute classes

## How to test

Run `python scripts/load_test_data.py` and ensure that users, topics, attributes are created and linked

## Quality Assurance

The following items have been updated:


## Screenshots

Nothing to show
